services/gateway/apis/http-version3/methods/blockAssets.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const blockAssetsSource = require('../../../sources/version3/blockAssets');
+const envelope = require('../../../sources/version3/mappings/stdEnvelope');
+
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/blocks/assets',
+	rpcMethod: 'get.blocks.assets',
+	tags: ['Blocks'],
+	params: {
+		blockID: { optional: true, type: 'string', min: 1, max: 64, pattern: /^([1-9]|[A-Fa-f0-9]){1,64}$/ },
+		height: { optional: true, type: 'string', min: 0, pattern: /([0-9]+|[0-9]+:[0-9]+)/ },
+		timestamp: { optional: true, type: 'string', min: 1, pattern: /([0-9]+|[0-9]+:[0-9]+)/ },
+		moduleID: { optional: true, type: 'string', min: 38, max: 41, pattern: /[0-9]+/ },
```

Comment:
Please confirm with the SDK/Research team to determine the max possible value for `moduleID` and update the `max` number of characters accordingly.
```suggestion
		moduleID: { optional: true, type: 'string', min: 1, max: 41, pattern: /[0-9]+/ },
```